atlasdb-client/src/main/java/com/palantir/atlasdb/table/api/AtlasDbImmutableTable.java

```diff
@@ -36,9 +36,21 @@
     Multimap<ROW, COLUMN_VALUE> getRowsMultimap(Iterable<ROW> rows);
     Multimap<ROW, COLUMN_VALUE> getRowsMultimap(Iterable<ROW> rows,
                                                 ColumnSelection columnSelection);
+    
+    /*
+     * This returns visitables for each row. It should be used if you want to visit a number of the first 
+     * matched columns instead of all the columns per row.
+     */ 
```

Comment:
Either include the args and the return value for a javadoc or use regular comments.